src/nccl_ofi_api.c

```diff
@@ -11,8 +11,8 @@
 
 _Static_assert(sizeof(nccl_net_ofi_conn_handle_t) <= NCCL_NET_HANDLE_MAXSIZE,
 	       "Size of OFI Handle is too large");
-_Static_assert(offsetof(nccl_net_ofi_conn_handle_t, state) <= NCCL_NET_HANDLE_MAXSIZE_V4,
-	       "Size of OFI Handle (without state) is too large");
+//_Static_assert(offsetof(nccl_net_ofi_conn_handle_t, state) <= NCCL_NET_HANDLE_MAXSIZE_V4,
```

Comment:
Wasn't sure why we still need this `Static_assert`. Do we need to deprecate support for plugin interface v4?

Reply:
Sorry, actually the only reason I needed to make the handle bigger was to add some debug info, so we shouldn't need to get rid of this assert at all. Will fix.